src/common/email/email.service.ts

```
@@ -0,0 +1,162 @@
+
+    if (!senderEmail) {
+      this.logger.error(
+        "SENDGRID_SENDER_EMAIL n'est pas défini dans la configuration.",
+
+        'EmailService',
+      );
+      throw new InternalServerErrorException("Adresse e-mail de l'expéditeur manquante.");
+    }
+
+    const expirationDate = new Date(
+      Date.now() + parseInt(process.env.INVITATION_TOKEN_EXPIRATION_MS || '86400000', 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)